modular_skyrat/master_files/code/modules/client/preferences/middleware/languages.dm

```diff
@@ -40,9 +40,7 @@
 
 /datum/preference_middleware/languages/apply_to_human(mob/living/carbon/human/target, datum/preferences/preferences) // SKYRAT EDIT CHANGE
```

Comment:
Since you're there, yeet this. We're in modular files.
```suggestion
/datum/preference_middleware/languages/apply_to_human(mob/living/carbon/human/target, datum/preferences/preferences)
```